Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/GBTLink.h

```diff
@@ -263,6 +263,11 @@ GBTLink::CollectedDataStatus GBTLink::collectROFCableData(const Mapping& chmap)
           printCalibrationWord(gbtC);
         }
         dataOffset += GBTPaddedWordLength;
+        //Adding calibration info in RU pointer
+        int calUser = gbtC->calibUserField;
+        ruPtr->calCount = gbtC->calibCounter;
+        ruPtr->nInj = calUser >> 16;
+        ruPtr->chargeInj = calUser & 0xff;
```

Comment:
To be sure: are these lines valid both for ITS and MFT?

Reply:
I can only speak for the case I now, i.e. how these fields are written in the raw data by the MFT RU firmware. I don't know how these fields are written to the raw data by ITS RU firmware. These are tested and validated with respect to the way the MFT fills these informations in the raw data.

Reply:
But now they are set in the code which is used both by ITS and MFT...
@iouribelikov who can answer this for ITS? If ITS uses calib info in other way, this part should be hidden in extra templated method specialized separately for ITS and MFT.

Reply:
@arakotoz @shahor02  *What* are we actually doing here ?   (What question should be propagated to the ITS experts ?)
Is it something related to injection runs (threshold scans) ?  The exact way the information about injected charge gets embedded into the raw data ?  
Or not ?

Meanwhile, I would agree with @shahor02 :  everything that is potentially MFT( or ITS)-specific should better go to a separate templated function.

Reply:
@iouribelikov apparently the MFT stores the info about the injection parameters in the GBT calib word, does the ITS do the same?